tests/zsa.rs

```diff
@@ -315,23 +332,28 @@ fn zsa_issue_and_transfer() {
     )
     .unwrap();
 
-    // 2. Split single ZSA note into 2 notes
-    let delta = 2; // arbitrary number for value manipulation
+    // 2. Split single ZSA note into 3 notes
+    let delta1 = 2; // arbitrary number for value manipulation
```

Comment:
let's stick to the existing style: `delta_1`

Reply:
all occurrences.

Reply:
Done